optashift-employee-rostering-webapp/pom.xml

```diff
@@ -14,6 +14,20 @@
 
   <name>Employee Rostering as a Service Webapp</name>
 
+  <properties>
+    <wildfly.version>11.0.0.Final</wildfly.version>
```

Comment:
All version properties should be declared in the parent pom.

Reply:
Thanks, will fix